src/components/interaction-designer/block-editors/SimplifiedNameEditor.vue

```diff
@@ -0,0 +1,94 @@
+<template>
```

Comment:
I created this new component instead of using the existing `NameEditor.vue` because there are many differences between the 02 desired UI. With this approach, it will be cleaner.